### DISCUSSION TITLE
Does MathComponents Support "limit" symbol?

### DISCUSSION THREAD

**Discussion by YaoKaiLun** [dolanmiu/docx#2505] [General]
I need to create the below formulas by Math Components, but i didn't see any constructor like "limitUpp" or "limitLow"
![image](https://github.com/dolanmiu/docx/assets/11460856/8643cd1e-ebac-46cc-96ea-1e29f7806c90)

![image](https://github.com/dolanmiu/docx/assets/11460856/19a945d9-3fbd-4631-b929-2052699bfd2a)

![image](https://github.com/dolanmiu/docx/assets/11460856/b7a9beed-f0ba-44d5-a931-d81635d1b391)

**Comment by dolanmiu** [comment]
Will be available once your PR is merged. Thanks!
